src/pydrex/core.py

```diff
@@ -140,30 +157,66 @@ class DefaultParams:
     Smaller grains will instead deform via grain boundary sliding,
     therefore not contributing to any further texture evolution.
 
+    .. note:: Values for this parameter do NOT correspond to a physical grain size
+        threshold. Grains in PyDRex are a surrogate numerical discretisation. There are
+        likely to be feedbacks between this parameter and `number_of_grains`. Values of
+        ~0.3 have been used in numerous studies which employed ~3000 grains.
+
     """
     nucleation_efficiency: float = 5.0
     """Dimensionless nucleation efficiency (λ*).
 
     This controls the nucleation of subgrains in `DeformationRegime.matrix_dislocation`.
 
+    The default value comes from [Kaminski & Ribe 2001](https://doi.org/10.1016%2Fs0012-821x%2801%2900356-9).
+
     """
     number_of_grains: int = 3500
-    """Number of surrogate grains for numerical discretisation of the aggregate."""
+    """Number of surrogate grains for numerical discretisation of the aggregate.
+
+    This is a numerical discretisation, so generally more grains is better. However,
+    there is a sharp tradeoff in performance, especially for M-index calculation.
+
+    """
     initial_olivine_fabric: MineralFabric = MineralFabric.olivine_A
-    """Olivine fabric (CRSS distribution) at the beginning of the simulation."""
-    disl_Peierls_stress: float = 10
-    """Stress barrier for activation of dislocation motion at low temperatures.
+    """Olivine fabric (CRSS distribution) at the beginning of the simulation.
 
-    .. note:: Not relevant if the unified dislocation creep flow law is used instead.
+    In general, the fabric should be allowed to change during the simulation,
+    but that is not yet implemented.
+
+    """
+    disl_Peierls_stress: float = 2
+    """Stress barrier in GPa for activation of dislocation motion at low temperatures.
+
+    - 2GPa suggested by [Demouchy et al. 2023](http://dx.doi.org/10.2138/gselements.19.3.151)
+
+    .. note:: Not relevant if the unified dislocation creep flow law is used.
 
     """
     # NOTE: For now, we just roll this into the prefactors.
     # water_fugacity: float = 1.5
     # """Approximate constant water fugacity of the aggregate (GPa)."""
+    # TODO: Check units of Garel 2020 pre-exp value below.
+    # TODO: Find and add references for the other value.
     disl_prefactors: tuple = (1e-16, 1e-17)
-    """Prefactors for dislocation creep exponential & power laws (s⁻¹)."""
-    diff_prefactor: float = 1e-10
-    """Prefactor for diffusion creep power law (s⁻¹)."""
+    """Prefactors for dislocation creep exponential & power laws (s⁻¹).
+
+    - B = 1.7e-16 s⁻¹ for the exponential law suggested by [Demouchy et al. 2023](http://dx.doi.org/10.2138/gselements.19.3.151)
+    - pre-exponential factor of 4.4e-17 Pa⁻ⁿs⁻¹ used for the exponential law by [Garel et al. 2020](http://dx.doi.org/10.1016/j.epsl.2020.116243)ᵃ
+
+    ᵃCheck units, why is the Pa⁻ⁿ in there...
```

Comment:
That is normal, it is coming from the general relationship between stress and strain.